osf/models/mixins.py

```diff
@@ -2141,22 +2141,22 @@ def suspend_spam_user(self, user, train_akismet=False):
         # Make public nodes private from this contributor
         for node in user.all_nodes:
             if self._id != node._id and len(node.contributors) == 1 and node.is_public and not node.is_quickfiles:
-                node.confirm_spam(save=True, train_akismet=train_akismet)
-                node.set_privacy('private', log=False, save=True)
+                node.confirm_spam(save=True)
```

Comment:
as stated elsewhere, not training akismet was intentional here, since we don't want to feed it bad info in the case that a user was over-zealously spammed.